src/blocks/remote-data-container/hooks/usePatterns.ts

```diff
@@ -76,6 +76,8 @@ export function usePatterns( remoteDataBlockName: string, rootClientId: string =
 			innerBlocks.push( createBlock( 'remote-data-blocks/pagination' ) );
 		}
 
+		innerBlocks.push( createBlock( 'remote-data-blocks/empty-result' ) );
```

Comment:
This ensures it's always available, and we can decide when to render it. If there's another place to insert this, let me know.

Reply:
This is the right place. 👍